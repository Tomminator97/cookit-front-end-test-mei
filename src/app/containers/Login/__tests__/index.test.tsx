import * as React from 'react';
import { Login } from '..';
import { createRenderer } from 'react-test-renderer/shallow';

const shallowRenderer = createRenderer();

describe('<Login />', () => {
  it('should render and match the snapshot', () => {
    shallowRenderer.render(<Login />);
    const renderedOutput = shallowRenderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
