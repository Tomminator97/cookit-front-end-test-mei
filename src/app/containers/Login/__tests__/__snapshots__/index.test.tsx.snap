// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> should render and match the snapshot 1`] = `
<React.Fragment>
  <Helmet
    defer={true}
    encodeSpecialCharacters={true}
  >
    <title />
    <meta
      content=""
      name="description"
    />
  </Helmet>
  <NavBar />
  <Login__Wrapper />
</React.Fragment>
`;
